tsconfig.json

```diff
@@ -13,6 +13,7 @@
     "noUnusedLocals": false,
     "jsx": "react",
     "jsxFactory": "JSX.createElement",
+    "typeRoots": [],
```

Comment:
Why is this necessary?

Reply:
Exclude parent directories from auomatic type detection